src/love.cpp

```diff
@@ -187,7 +187,7 @@ static DoneAction runlove(int argc, char **argv, int &retval)
 			lua_rawseti(L, -2, -2);
 		}
 
-		lua_pushstring(L, "embedded boot.lua");
+		lua_pushstring(L, "[love \"boot.lua\"]");
```

Comment:
I did a quick github code search and I see at least some projects/libraries that check for `"embedded boot.lua"` in the args list and do stuff when it's found, so IMO we should change this line for 12.0 rather than 11.4.